### PR TITLE
HTML fixes

### PR DESCRIPTION
**Pull request by CyberShadow**

### CODE REVIEW

File: std/datetime.d [hunk 1]
```diff
@@ -28872,17 +28872,17 @@ version(StdDdoc)
         occur (especially for historical dates). Also, the TZ Database files
         include far more time zones than Windows does. So, for accurate
         time zone information, use the TZ Database files with
-        $(LREF PosixTimeZone) rather than $(LREF WindowsTimeZone). However, because
-        $(LREF WindowsTimeZone) uses Windows system calls to deal with the time,
+        $(LREF PosixTimeZone) rather than $(D WindowsTimeZone). However, because
```
Comment (JackStouffer) [inline std/datetime.d]: Is `$(LREF PosixTimeZone)` intentionally not `$(D PosixTimeZone)` here?

File: std/internal/math/errorfunction.d [hunk 2]
```diff
@@ -16,7 +16,7 @@
  *  POWER = $1<sup>$2</sup>
  *  BIGSUM = $(BIG &Sigma; <sup>$2</sup><sub>$(SMALL $1)</sub>)
  *  CHOOSE = $(BIG &#40;) <sup>$(SMALL $1)</sup><sub>$(SMALL $2)</sub> $(BIG &#41;)
- *  TABLE_SV = <table border=1 cellpadding=4 cellspacing=0>
+ *  TABLE_SV = <table border="1" cellpadding="4" cellspacing="0">
```
Comment (JackStouffer) [inline std/internal/math/errorfunction.d]: This isn't really necessary. In HTML5 they can be either.

File: std/internal/math/gammafunction.d [hunk 3]
```diff
@@ -8,7 +8,7 @@
  *
  *
 Macros:
- *  TABLE_SV = <table border=1 cellpadding=4 cellspacing=0>
+ *  TABLE_SV = <table border="1" cellpadding="4" cellspacing="0">
```
Comment (JackStouffer) [inline std/internal/math/gammafunction.d]: ditto

File: std/datetime.d (hunk 1 above)
Comment (CyberShadow) [inline std/datetime.d]: Yes

File: std/internal/math/errorfunction.d (hunk 2 above)
Comment (CyberShadow) [inline std/internal/math/errorfunction.d]: http://www.webdirections.org/blog/five-reasons-why-you-should-quote-attribute-values-in-html5/

I need these changes merged so we can have automatic HTML validation for PRs.

**Comment by JackStouffer** [comment]
LGTM

**Comment by CyberShadow** [comment]
Other PRs:

https://github.com/D-Programming-Language/dlang.org/pull/1134
https://github.com/D-Programming-Language/druntime/pull/1416

**Pull request merged**